model-engine/model_engine_server/inference/batch_inference/vllm_batch.py

```diff
@@ -264,6 +264,10 @@ def tool_func(text: str, past_context: Optional[str]):
                 gen_item.completed = True
                 continue
 
+            if gen_item.remaining_tokens <= 0:
```

Comment:
nit: we can just check for this in the prev conditional

Reply:
sure